packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts

```diff
@@ -321,31 +321,34 @@ describeCompat("Message size", "NoCompat", (getTestObjectProvider, apis) => {
 			},
 		).timeout(chunkingBatchesTimeoutMs);
 
-		describe(`Large payloads (exceeding the 1MB limit) - ${
+		describe(`Large payloads - ${
 			enableGroupedBatching ? "grouped" : "regular"
 		} batches`, () => {
 			describe("Chunking compressed batches", () =>
 				[
-					{ messagesInBatch: 1, messageSize: 2 * 1024 * 1024 }, // One large message
-					{ messagesInBatch: 3, messageSize: 2 * 1024 * 1024 }, // Three large messages
-					{ messagesInBatch: 1500, messageSize: 4 * 1024 }, // Many small messages
+					{ messagesInBatch: 1, messageSize: 51 * 1024 }, // One large message (51 KB each)
```

Comment:
consider defining a const kb = 1024 at the top once and re-use across the test suite. Will make it more readable.

Reply:
What was the resolution here?

Reply:
Added a const `bytesPerKB`